frontend/src/store/modules/genes.js

```diff
@@ -5,13 +5,22 @@ const data = {
 };
 
 const getters = {
-  geneName: state => state.gene.geneName,
+  geneName: state => state.gene.id,
 };
 
 const actions = {
   async getGeneData({ commit }, { model, id }) {
-    const gene = await genesApi.fetchGeneData(model, id);
+    console.warn(`TODO: use model: ${model}`);
```

Comment:
Because of linter settings, `console.log` throws an error in production mode so the Docker container doesn't build. This is allowed locally, but extra care should be given to remove them.

Reply:
Have added a Jira task about this now. Technically this frontend stuff shouldn't be in this branch and PR but I think we can ignore this for now.